editor/inserter/menu.js

```diff
@@ -249,7 +242,11 @@ class InserterMenu extends Component {
 		this.changeMenuSelection( 'search' );
 	}
 
-	getBlockItem( block ) {
+	switchTab( tab ) {
+		this.setState( { tab: tab } );
```

Comment:
Minor: ES2015 includes an object shorthand convenience when key and value are the same:

```js
this.setState( { tab } );
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015